rayguiexample/source/app.d

```diff
@@ -103,7 +103,7 @@ int main() {
     //----------------------------------------------------------------------------------
 
     // Custom GUI font loading
-    //Font font = LoadFontEx("fonts/rainyhearts16.ttf", 12, 0, 0);
+    //Font font = LoadFontEx("fonts/rainyhearts16.ttf".toStringz, 12, 0, 0);
```

Comment:
D string literals are compatible with C strings. This line works as-is in D.

Reply:
I have found that sometimes it works, and sometimes it doesn't. As an experiment, I just tried removing all the `.toStringz` uses in my code, and the only errors were for the functions `MeasureTextEx` and `DrawTextEx`. Having experienced this error before, I had added this use of `toStringz` for peace of mind, thinking that it might just suddenly stop working one day without it.

Do you know why using a D string works for some Raylib functions but not for others?

Reply:
I just figured it out. It works without `toStringz` when the string is known at compile-time, but fails when a variable is used. I will now keep this in mind.

Reply:
Yes, any *string literals* are implicitly convertible to `const char *` and will have proper zero termination. That is a part of the D language specification: https://dlang.org/spec/expression.html#string_literals

So please remove this change.